### ISSUE TITLE
Many archived grammars after Atom sunset

### ISSUE THREAD

**Issue by Visne** [github/linguist#6251]
I assume you are already aware of this, but I couldn't find an issue for it yet. On December 15th 2022 all Atom repositories were archived. This means that Linguist now depends on a bunch of archived repositories. They should probably be replaced (or unarchived).

The following Atom repositories are currently used:

- [ ] https://github.com/atom/language-clojure
- [ ] https://github.com/atom/language-coffee-script
- [ ] https://github.com/atom/language-gfm
- [ ] https://github.com/atom/language-html
- [ ] https://github.com/atom/language-javascript
- [ ] https://github.com/atom/language-less
- [ ] https://github.com/atom/language-property-list
- [ ] https://github.com/atom/language-ruby
- [ ] https://github.com/atom/language-sass
- [ ] https://github.com/atom/language-shellscript
- [ ] https://github.com/atom/language-yaml

I found these with the following command:
```fish
wget -O - -o /dev/null https://raw.githubusercontent.com/github/linguist/master/vendor/README.md | grep https://github.com/atom/ | cut -d \( -f 2 | cut -d \) -f 1 | sort | uniq
```

**Comment by Alhadis** [comment]
> They should probably be replaced

Why?

**Comment by Visne** [comment]
Because they are unmaintained, and unchangeable.

**Comment by Alhadis** [comment]
@Visne If they serve their purpose, and there're no inaccuracies with their highlighting, then there's no immediate need to remove them. If an update or correction is required, _then_ we can look into replace the offending grammar(s).

**Comment by Visne** [comment]
I never said remove, I said replace or unarchive. Forking could also be possible.

**Comment by lildude** [comment]
We're well aware that these repos have been archived and may need replacements at some point in the future. However we're not going to change things for the sake of changing things. Many of these grammars have been stable and unchanged without major problems for quite some time so replacing them now without good reason other than they're no longer maintained risks introducing unnecessary changes/breakages. Case it point: we did precisely this with the Svelte grammar which was changed from an old unmaintained grammar to a newer one and it resulted in worse syntax highlighting, not better. See https://github.com/github/linguist/discussions/6164 for all the gory details.

In short, if it ain't broke, don't change it.

GitHub has no immediate plans to delete these archived repos so there's little risk of them disappearing on us unannounced.

We will assess each grammar as problems arise and if a better grammar comes along that fixes a known issue without breaking anything else, we will replace it.

As such, I don't think we need to keep this issue open. Thank-you for showing your interest and concern on this matter.

**Comment by Visne** [comment]
Alright, I understand your reasoning. I'm wondering if it's okay to open issues related to those grammars in this repo.

**Comment by Alhadis** [comment]
I've opened a discussion proposing a feature related to the OP. See #6327.
